cmd/cluster-agent/app/app.go

```diff
@@ -354,6 +354,7 @@ func start(cmd *cobra.Command, args []string) error {
 			server := admissioncmd.NewServer()
 			server.Register(config.Datadog.GetString("admission_controller.inject_config.endpoint"), mutate.InjectConfig, apiCl.DynamicCl)
 			server.Register(config.Datadog.GetString("admission_controller.inject_tags.endpoint"), mutate.InjectTags, apiCl.DynamicCl)
+			server.Register(config.Datadog.GetString("admission_controller.auto_instru.endpoint"), mutate.InjectAutoInstru, apiCl.DynamicCl)
```

Comment:
IMO, it is better to not shrink option name if don't have a technical restriction

```suggestion
			server.Register(config.Datadog.GetString("admission_controller.auto_instrumentation.endpoint"), mutate.InjectAutoInstrumentation, apiCl.DynamicCl)
```

Reply:
I'm just avoiding super long config names and env vars i.e `DD_ADMISSION_CONTROLLER_AUTO_INSTRUMENTATION_ENDPOINT`. I do agree it's clearer but prone to typos